github/ci/prow-deploy/files/orgs.yaml

```diff
@@ -716,9 +717,12 @@ orgs:
           - dhiller
           - rmohr
           - fgimenez
+          - brianmcarey
         members:
           - phoracek
         privacy: closed
+        repos:
+          secrets: admin
```

Comment:
:heart:  very good idea to add access for everyone in the task force here. I guess you took `admin` to enable us giving other persons access to the repo manually?

Reply:
yes exactly, maybe it could be `write` only so that everything is controlled by peribolos, wdyt?